src/SqsPoller/SqsPollerHostedService.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace SqsPoller
 {
-    internal class SqsPollerHostedService: BackgroundService
+    public class SqsPollerHostedService: BackgroundService
```

Comment:
what's the main reason to make it public? I do think it will violate  main principle of the library

Reply:
well, the main difference between library and framework is in restriction. If library restricts you - it's framework. Are you sure that it's a library? Let's discuss it on monday